sunpy/net/attrs.py

```diff
@@ -28,13 +28,15 @@
 from .jsoc import attrs as jsoc
 from .dataretriever.attrs import goes
 
+from .vso.attrs import Physobs
 from ._attrs import Time, Instrument, Wavelength, Level, Sample, Detector, Resolution
 from .base_client import BaseClient
 
 # Trick the docs into thinking these attrs are defined in here.
-for _a in (Time, Instrument, Wavelength, Level, Sample, Detector, Resolution):
+for _a in (Time, Instrument, Wavelength, Level, Sample, Detector, Resolution,
+           Physobs):
     _a.__module__ = __name__
 
 
 __all__ = ['Time', 'Instrument', 'Wavelength', 'Level', 'Sample', 'Detector', 'Resolution',
-           'vso', 'jsoc', 'goes']
+           'vso', 'jsoc', 'goes', 'Physobs']
```

Comment:
Why have has `Physobs` been added here? I think this might be why the doc build currently isn't working.

Reply:
I think this might have been solved in Fido refactoring PR by @cadair, I have to check,